opentelemetry-proto/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## vNext
 
-- Bump MSRV to 1.70 [#2179](https://github.com/open-telemetry/opentelemetry-rust/pull/2179)
+## v0.26.1
```

Comment:
I am trying to avoid patches unless really needed. if this is blocking tracing integration, then lets do the patch release.

Reply:
I am blocking the tracing integration because this is just broken and I don't want to deal with users complaining about it.

Why don't you want to do patch releases?

Reply:
> Why don't you want to do patch releases?

Don't want to do it unless it is really something that should be patched. Like a regression in the last release or security hot fix etc.

We are doing a release every ~3 weeks now for next few months (until we reach stable). Any normal changes can be shipped as part of the regular release itself.

If this is a blocker for `tracing`, happy to make an exception and do the patch.